### DISCUSSION TITLE
App Menu Grayed Out

### DISCUSSION THREAD

**Discussion by michaelmcdonald** [ttscoff/bunch-docs#133] [Q&A] [answered]
Was playing around with some bunches, started getting an error about not being able to open an application. Had to end for the day so I quit my apps / Bunch. Came back this morning and now Bunch will open, a window will ever so quickly flash (I can't see what it says), and then the menu in Bunch is all sorts of wrong (doesn't list my bunches and is grayed out). Can't even get to the log (and, curiously enough: I can't find documentation anywhere on where the log location actually is aside from how to get to it using the Help menu). 

What did I break? I have tried deleting the app / re-install to no avail. I have moved the bunches I made out of the directory and left just the example bunch. Did I kill it?

![Screen Shot 2021-08-11 at 07 29 01](https://user-images.githubusercontent.com/4342041/129023560-d9157cdb-ca23-46b4-b1fb-0673efd1f69e.png)

![Screen Shot 2021-08-11 at 07 28 37](https://user-images.githubusercontent.com/4342041/129023561-10e4fe31-9de1-4eeb-aad3-c74f43b02317.png)

**Comment by ttscoff** [comment]
Apparently you did kill it :P. I've never seen this happen before. 
Finding the log may or may not be helpful, but you can locate them at: 
~/Library/Logs/Bunch (or BunchBeta). Let me know if you find anything 
pertinent there.

Thanks,
Brett

**Comment by michaelmcdonald** [comment]
Log file is empty :( The two preceding it are empty as well:

![Screen Shot 2021-08-11 at 11 52 23](https://user-images.githubusercontent.com/4342041/129062144-60d6b56c-b855-4dd2-93a3-333e8118e51c.png)

Thoughts on how to completely remove it so I can install it fresh? Or how to increase the logging verbosity without doing so via the app? lol

**Comment by ttscoff** [comment, marked as answer]
To remove completely, be sure to also delete the preference file located 
at `~/Library/Preferences/com.brettterpstra.bunch`. I would also delete 
the entire Bunch Folder and let Bunch re-create it when you re-install 
(just move your existing Bunches to a different folder). Reboot before 
re-installing Bunch make sure preferences are cleared from cache.

Like I said, I haven't seen this happen before, so I don't have a less 
"nuclear" solution to offer.

-Brett

**Reply by michaelmcdonald** [reply]
Went with the nuclear option and was able to re-install the Beta. Everything appears to be working now!
